docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -399,19 +397,19 @@ The observations need to relate to some results of the simulation, so that the
 algorithms can compare them. We have some observations from the polynomial that
 were measured at the points 0, 2, 4, 6 and 8. The indices here happen to align
 with the x values of the polynomial evaluation, but this is incidental. The
-indices says where the observations in the file match the result indices. Put
+indices say where the observations in the file match the result indices. Put
 the following observations in the file ``poly_obs_data.txt``:
 
     .. literalinclude:: with_observations/poly_obs_data.txt
 
-The observations are written one for each line, with the first number
+There is one observations per line, with the first number
 signifying the observed value, and the second number signifying the uncertainty.
 
 Now let's describe the observations we have to ERT. This is done with the
```

Comment:
"we have to ERT", another typo that you can fix :). ERT is not yet a verb, maybe later..

Reply:
ERT is the subject here :)
We are describing the observations to ERT - ERT is to whom we are describing the observations! 
But yeah, I'll try and come up with a better sentence.

Reply:
I've pushed a new proposal.